packages/config/src/arbitrum-testnet.json

```diff
@@ -38,28 +38,30 @@
   "extra_life_start_block": 9265867,
   "keys_address": "0x0000000000000000000000000000000000000000",
   "keys_start_block": 9265867,
-  "smol_brains_address": "0x5754eaf59f04e8ad6cde93de13c0224e95db64eb",
-  "smol_brains_start_block": 9148080,
-  "smol_brains_land_address": "0x0000000000000000000000000000000000000000",
-  "smol_brains_land_start_block": 9265867,
+  "smol_brains_address": "0x2542421ACA04A98f5Cf04DA97a36DAD8F1FaC3f4",
+  "smol_brains_start_block": 9382049,
+  "smol_brains_land_address": "0xF8e004f5447a3D9ae0841fAc98523Fa89c1Ea63d",
+  "smol_brains_land_start_block": 9382088,
   "smol_brains_pets_address": "0x0000000000000000000000000000000000000000",
   "smol_brains_pets_start_block": 9265867,
   "smol_brains_school_address": "0x0000000000000000000000000000000000000000",
   "smol_brains_school_start_block": 9265867,
   "smol_cars_address": "0x0000000000000000000000000000000000000000",
   "smol_cars_start_block": 9265867,
-  "smol_farm_address": "0xfE8c1ac365bA6780AEc5a985D989b327C27670A1",
-  "smol_farm_start_block": 9141924,
+  "smol_farm_address": "0x5b18510C5f6a00c3b8Cd5479a0b791E8950C36E8",
```

Comment:
Does this point to the proxy or the implementation address?

Reply:
This is the proxy